### PR TITLE
Complete UI for Showtimes page

### PR DESCRIPTION
**Pull request by HuyenNgo**
## To have UI for Showtimes page:
- Correct UI
- Correct convention

### CODE REVIEW

File: src/components/schedualPage/infoFilm.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+          {/* Finish row Info */}
+           {/* Start row Info */}
+          <div className="row rowLeftInfo">
+            <p className="contentTitle">Đạo diễn</p>
+            <p className="contentInfo ng-binding"> Jordan Peele </p>
+          </div>
+          {/* Finish row Info */}
+          <div className="row rowLeftInfo">
+            <p className="contentTitle">Diễn viên</p>
+            <p className="contentInfo ng-binding">Winston Duke, Lupita Nyong'o</p>
```
Comment (tvc12) [inline src/components/schedualPage/infoFilm.js]: ## Missing wrap '. May be use \\'

File: src/App.js [hunk 2]
```diff
@@ -20,7 +20,7 @@ class App extends Component {
             <Route exact path="/" component={HomePage} />
             <Route path='/movies' component={MoviesPage} />
             <Route path='/contact' component={ContactPage} />
-            <Route path='*' component={NotFoundPage} />
+            <Route path='*' component={SchedualPage} />
```
Comment (tvc12) [inline src/App.js]: ## Please don't remove NotFoundPage route.
- You can create a new route.
- You have to rename SchedualPage -> ShowTime

File: public/index.html [hunk 3]
```diff
@@ -30,6 +30,9 @@
     <link rel="stylesheet" type="text/css" href="%PUBLIC_URL%/css/owl.carousel.min.css" />
     <link rel="stylesheet" type="text/css" href="%PUBLIC_URL%/css/style.css" />
     <link rel="stylesheet" type="text/css" href="%PUBLIC_URL%/css/responsive.css" />
+    <link rel="stylesheet" type="text/css" href="%PUBLIC_URL%/revolution/css/colorbox.css" />
+    <link rel="stylesheet" type="text/css" href="%PUBLIC_URL%/revolution/css/custom-schedual.css" />
+    <link rel="stylesheet" type="text/css" href="%PUBLIC_URL%/revolution/css/schedualPage.css" />
```
Comment (cuongtk8) [inline public/index.html]: Please import `.css file` from `.js` file.

**Review by cuongtk8 — CHANGES_REQUESTED** [review]

File: public/revolution/css/schedualPage.css [hunk 4]
```diff
@@ -0,0 +1,195 @@
+.detailContentCard{
```
Comment (cuongtk8) [inline public/revolution/css/schedualPage.css]: This file should locate into `src/styles`.

File: src/App.js [hunk 5]
```diff
@@ -8,7 +8,7 @@ import {
   ForgetPassword,
   BackToTop
 } from './components';
-import { HomePage, MoviesPage, NotFoundPage, ContactPage } from './pages';
+import { HomePage, MoviesPage, NotFoundPage, ContactPage,SchedualPage } from './pages';
```
Comment (cuongtk8) [inline src/App.js]: Please add a space after comma.

File: src/pages/SchedualPage.js [hunk 6]
```diff
@@ -0,0 +1,26 @@
+import React, { Component } from 'react'
```
Comment (cuongtk8) [inline src/pages/SchedualPage.js]: Please replace `SchedualPage` by `MovieShowtimes`.

File: src/pages/index.js [hunk 7]
```diff
@@ -2,3 +2,5 @@ export { default as HomePage } from './HomePage';
 export { default as NotFoundPage } from './NotFoundPage';
 export { default as MoviesPage } from './MoviesPage';
 export { default as ContactPage } from './ContactPage';
+export { default as SchedualPage } from './SchedualPage';
```
Comment (cuongtk8) [inline src/pages/index.js]: Please replace `SchedualPage` by `MovieShowtimes`.

File: src/components/schedualPage/top.js [hunk 8]
```diff
@@ -0,0 +1,83 @@
+import React, { Component } from 'react'
```
Comment (cuongtk8) [inline src/components/schedualPage/top.js]: Please replace `SchedualPage` folder by `MovieShowtimes` folder.

File: src/components/schedualPage/index.js [hunk 9]
```diff
@@ -0,0 +1,3 @@
+export {default as Top} from './top';
+export {default as InfoFilm} from './infoFilm';
+export {default as Schedual } from './schedual';
```
Comment (cuongtk8) [inline src/components/schedualPage/index.js]: Please replace `SchedualPage` folder by `MovieShowtimes` folder.

File: src/components/index.js [hunk 10]
```diff
@@ -1,3 +1,4 @@
 export * from './common';
 export * from './homePage';
 export * from './moviesPage';
+export * from './schedualPage';
```
Comment (cuongtk8) [inline src/components/index.js]: Please replace `SchedualPage` by `MovieShowtimes`.

File: src/App.js (hunk 2 above)
Comment (cuongtk8) [inline src/App.js]: Please replace `SchedualPage` by `MovieShowtimes`.

File: src/App.js (hunk 5 above)
Comment (cuongtk8) [inline src/App.js]: Please replace `SchedualPage` by `MovieShowtimes`.

File: public/revolution/css/schedualPage.css [hunk 11]
```diff
@@ -0,0 +1,195 @@
...
+    float: left;
+    border-top: 1px solid #ebebec;
+    border-bottom: 1px solid #ebebec;
+    border-left: 1px solid #ebebec;
+    border-top-left-radius: 4px;
+    border-bottom-left-radius: 4px;
+    width: 28%;
+}
+.center{
+
```
Comment (cuongtk8) [inline public/revolution/css/schedualPage.css]: Please format code for this file.

File: src/components/schedualPage/infoFilm.js [hunk 12]
```diff
@@ -0,0 +1,69 @@
+import React, { Component } from 'react'
```
Comment (cuongtk8) [inline src/components/schedualPage/infoFilm.js]: Please format code for this file.

File: src/components/schedualPage/schedual.js [hunk 13]
```diff
@@ -0,0 +1,279 @@
...
+
+                                  </dd>
+
+                                </dl>
+                               {/* Start Show InFo Of Theater */}
+                              </div>
+
+                            </dd>
+
+
```
Comment (cuongtk8) [inline src/components/schedualPage/schedual.js]: Please format code for this file.

File: src/components/schedualPage/top.js [hunk 14]
```diff
@@ -0,0 +1,83 @@
...
+              <Schedual />
+            </div>
+          </div>
+        </div>
+        {/* Finish Modal Show Schedual Film */}
+
+        {/* Finish Top */}
+      </div>
+
+
```
Comment (cuongtk8) [inline src/components/schedualPage/top.js]: Format code for this file.

**Review by cuongtk8 — CHANGES_REQUESTED** [review]

**Comment by cuongtk8** [comment]
@HuyenNgo, please help me fix all issues!

**Pull request merged**